config/clusters/cloudbank/humboldt.values.yaml

```diff
@@ -39,23 +39,16 @@ jupyterhub:
       CILogonOAuthenticator:
         oauth_callback_url: https://humboldt.cloudbank.2i2c.cloud/hub/oauth_callback
         allowed_idps:
-          http://google.com/accounts/o8/id:
-            username_derivation:
-              username_claim: "email"
-            allowed_domains:
-              - "2i2c.org"
-              - "berkeley.edu"
-              - "humboldt.edu"
           https://sso.humboldt.edu/idp/metadata:
             username_derivation:
               username_claim: "email"
             allow_all: true
-          urn:mace:incommon:berkeley.edu:
+          http://google.com/accounts/o8/id:
             username_derivation:
               username_claim: "email"
-            allow_all: true
+            allowed_domains:
+              - humboldt.edu
```

Comment:
Here we can see also that I ordered the google idp used for admin access below the https://sso.humboldt.edu/idp/metadata idp. This is in preparation for https://github.com/jupyterhub/oauthenticator/issues/690 that could help us ensure the non-admin idp is pre-selected by default rather than the admin-only relevant idp.